src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -361,26 +388,22 @@ void ProbeDirectory(string manifestDirectory, string featureBand)
                     .Where(dir => File.Exists(Path.Combine(dir, "WorkloadManifest.json")))
                     .Select(dir =>
                     {
-                        ReleaseVersion? releaseVersion = null;
-                        ReleaseVersion.TryParse(Path.GetFileName(dir), out releaseVersion);
-                        return (directory: dir, version: releaseVersion);
-                    })
-                    .Where(t => t.version != null)
-                    .OrderByDescending(t => t.version)
-                    .ToList();
+                        return (directory: dir, version: Path.GetFileName(dir));
```

Comment:
why is the above no longer required?

Reply:
This whole list was only used for one purpose: to find the latest version and return that. This isn't a very efficient way of doing that. That logic is now built into line 397 that finds the latest version and returns the directory plus its version as a string. It turned out that we don't even need the version as a Version, so I can just leave it as a string named version.